docs/charts/index.yaml

```diff
@@ -33,7 +49,7 @@ entries:
     version: 0.3.0
   - apiVersion: v2
     appVersion: 1.8.2
-    created: "2021-06-02T23:05:43.379448+08:00"
+    created: "2021-09-09T09:24:18.943465+08:00"
```

Comment:
the job will change all version's created time. We should rollback?

Reply:
right, I have rolled back other changes.